README.md

```diff
@@ -1 +1,57 @@
-# Hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```

Comment:
- [ ] I see that you used the correct template! 🚀 However, please either use your own logo if you have one or remove this Logo section! 🤓 🔧